HW_03/Task1/Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Threading;
+
+namespace Task1
+{
+    class Program
+    {
+        private static bool bRepeateFlag = true;
```

Comment:
bRepeateFlag - strange name

Reply:
There is a reason for such name. I use this variable as a flag variable to determine whether the program should be repeated or could be terminated. 'b' in its name means 'bool'. Considering the overall design of this program I don't see a better name for this variable.